src/plugins/data/common/field_formats/converters/numeral.ts

```diff
@@ -56,11 +56,14 @@ export abstract class NumeralFormat extends FieldFormat {
   protected getConvertedValue(val: any): string {
     if (val === -Infinity) return '-∞';
     if (val === +Infinity) return '+∞';
-    if (typeof val !== 'number') {
+
+    const isBigInt = typeof val === 'bigint';
```

Comment:
Does it make sense to add a unit test for bigint?

Reply:
On their way.